test/unittests/xmllib/models/test_root.py

```diff
@@ -0,0 +1,39 @@
+from dsp_tools.xmllib.models.config_options import Permissions
+from dsp_tools.xmllib.models.dsp_base_resources import AnnotationResource
+from dsp_tools.xmllib.models.resource import Resource
+from dsp_tools.xmllib.models.root import XMLRoot
+
+
+def test_root_add_resources() -> None:
+    xml_root = XMLRoot.create_new("0000", "test")
+
+    # add_resource
+    anno_res = AnnotationResource("annoID", "label", "id1", ["comment"], Permissions.OPEN)
+    anno_res.add_comment("comment2")
+    xml_root.add_resource(anno_res)
+    assert len(xml_root.resources) == 1
+
+    # add_resource_multiple
+    many_resources = []
+    res1 = Resource.create_new("id1", ":Restype", "label")
+    res1.add_bool(":boolProp", True)
```

Comment:
why do you add a value, but then you don't test for its presence? Then it would be simpler not to add the value, to keep the test concise.

Reply:
Because that is extensively tested for the resource. I could actually remove it since we dont require a value.